### PR TITLE
DOC: Update Introduction sections in Python Notebooks

### PR DESCRIPTION
**Pull request by shaneahmed**
- Update and streamline example notebooks introduction sections especially related to installation instructions.

### CODE REVIEW

File: examples/01_example_wsiread.ipynb [hunk 1]
```diff
@@ -21,20 +21,40 @@
     "id": "p3l08vukKAWo"
```
Comment (DavidBAEpstein) [inline examples/01_example_wsiread.ipynb:21]: <p><u>_on the right of the Kaggle screen, 3) log in to your Kaggle account.</u></p><p>Add 4) tick "Internet" checkbox under Settings, to enable necessary downloads.</p><p> </p><p><em>The stray underscore was intended (by Mostafa I think) to help set this paragraph in italics. I agree with the italics.</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (DavidBAEpstein) [inline examples/01_example_wsiread.ipynb:21]: <p> </p><p> </p><p><u>Check that "colab" appears in the browser address bar after you click on "Open in Colab".</u></p><p>Check that "colab" or "kaggle", as appropriate, appear in the address bar.</p><p> </p><p><u>on either Github or colab</u></p><p>on Github, Colab or Kaggle</p><p> </p><p><u>locally or  outside Colab</u></p><p>on any platform, except for Colab or Kaggle,</p><p> </p><p><em>Remarks: 1) The user may have access to machines that run notebooks remotely, so "locally" is misleading. 2) Advice to use pip with --ignore is only available in installation.rst. It should appear in README.</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (DavidBAEpstein) [inline examples/01_example_wsiread.ipynb:21]: <p><em>I'm sorry that I have been so slow on the tissuemask notebook. While I was working on that notebook, I decided that the following documentation (with appropriate deletions in the code cell) would be less puzzling to a naive user:</em></p><p> </p><p>This cell should be skipped, unless you are using Colab or Kaggle. If you are using either Colab or Kaggle, this cell prepares the Python environment for<code>tiatoolbox</code>, which may give rise to harmless error messages: please ignore them.</p><p> </p><p>Outside Colab or Kaggle, the notebook expects&nbsp;<code>tiatoolbox</code>to already be installed. (See the instructions in&nbsp;<a href="https://github.com/TIA-Lab/tiatoolbox/blob/master/README.md#install-python-package" target="_blank">README</a>.)</p><p><em>If we decide to do this, then the next code cell should be deleted, which is an improvement on my earlier approach.</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (DavidBAEpstein) [inline examples/01_example_wsiread.ipynb:21]: <p><u>The next cell installs tiatoolbox including any pre-requisite software.</u></p><p>The next cell installs tiatoolbox, including prerequisite software, oni Colab or Kaggle, but should be skipped on other platforms.</p><p><em>If you accept my suggestions, the documentation is now repetitive, and the repetition should be eliminated.</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (DavidBAEpstein) [reply examples/01_example_wsiread.ipynb:21]: <p>Italics still not right. Move underscore to the end, after "downloads."</p>

File: examples/01_example_wsiread.ipynb [hunk 2]
```diff
@@ -12,7 +12,7 @@
     "\n",
```
Comment (DavidBAEpstein) [inline examples/01_example_wsiread.ipynb:12]: <p>The text cell above should be deleted, and the code cell below should be deleted.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (DavidBAEpstein) [inline examples/01_example_wsiread.ipynb:12]: <p>The next cell should be SKIPPED on all platforms, except for Colab or Kaggle, where it installs<code>tiatoolbox</code>, and other prerequisite software. Harmless error messages should be ignored. Outside Colab or Kaggle, the notebook expects  <code>tiatoolbox</code> to already be installed. (See the instructions in <a href="https://github.com/TIA-Lab/tiatoolbox/blob/master/README.md#install-python-package" target="_blank">README</a>.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (shaneahmed) [reply examples/01_example_wsiread.ipynb:12]: Addressed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/132/commits/f63e9bef28deb14b8e74a62bb9df12806ebceedc

Comment (shaneahmed) [reply examples/01_example_wsiread.ipynb:12]: Addressed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/132/commits/f63e9bef28deb14b8e74a62bb9df12806ebceedc

File: examples/01_example_wsiread.ipynb (hunk 1 above)
Comment (shaneahmed) [reply examples/01_example_wsiread.ipynb:21]: Addressed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/132/commits/0a7e3eef1bcb98b1137180859fd0cdeb8769d9e9

Comment (shaneahmed) [reply examples/01_example_wsiread.ipynb:21]: Addressed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/132/commits/0a7e3eef1bcb98b1137180859fd0cdeb8769d9e9

Comment (shaneahmed) [reply examples/01_example_wsiread.ipynb:21]: Addressed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/132/commits/0a7e3eef1bcb98b1137180859fd0cdeb8769d9e9

Comment (shaneahmed) [reply examples/01_example_wsiread.ipynb:21]: Addressed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/132/commits/0a7e3eef1bcb98b1137180859fd0cdeb8769d9e9

File: examples/04_example_patchextraction.ipynb [hunk 3]
```diff
@@ -36,7 +36,7 @@
         "\n",
```
Comment (John-P) [inline examples/04_example_patchextraction.ipynb:36]: <p>Weird line break here:</p><p> </p><p>Th</p><p>e first line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

File: examples/05_example_patchpredictor.ipynb [hunk 4]
```diff
@@ -375,7 +375,7 @@
         "\n",
```
Comment (John-P) [inline examples/05_example_patchpredictor.ipynb:375]: <p>This cell does not appear to be explained.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

File: examples/04_example_patchextraction.ipynb (hunk 3 above)
Comment (shaneahmed) [reply examples/04_example_patchextraction.ipynb:36]: <p>It's probably an issue related to ReviewNB not with the notebook itself. I can see it fine on my notebook locally. Please let me know if there is still an issue.</p>

**Comment by DavidBAEpstein** [comment]
1) I don't understand why there is no mention here of the work I did today on ReviewNB. 
2) I have made one or two comments in the initial cells. These were all in reference to the wsireader notebook, but they are equally relevant to the other notebooks. I also made a few very minor comments about the wsireader notebook that are not relevant to the other notebooks, but they could be dealt with now, even though the do not relate to the introductory sections.
3) Sometime ago, I created a branch for the tissuemask notebook. I'm planning to copy the cells from wsireader into tissuemask as soon as I find out that Shan has implemented the changes I suggest in the introductory sections of wsireader.
4) PR#132 should be merged with develop after my comments are dealt with. I still don't see any way of making this into an official approval by a reviewer to PR#132.

**Review by DavidBAEpstein — CHANGES_REQUESTED** [review]
See my contribution via ReviewNB

**Review by DavidBAEpstein — CHANGES_REQUESTED** [review]
Because I had problems with ReviewNB, I have reviewed each notebook in a separate file,
using vim rather than Github facilities. Please let me know if this method is working OK.

README.md is fine
[20210811_David_review_01wsi.txt](https://github.com/TIA-Lab/tiatoolbox/files/6976586/20210811_David_review_01wsi.txt)
[20210812_David_review_02stain.txt](https://github.com/TIA-Lab/tiatoolbox/files/6976587/20210812_David_review_02stain.txt)
[20210812_David_review_04patchextraction.txt](https://github.com/TIA-Lab/tiatoolbox/files/6976588/20210812_David_review_04patchextraction.txt)
[20210812_David_review_05predict.txt](https://github.com/TIA-Lab/tiatoolbox/files/6976589/20210812_David_review_05predict.txt)

The tissuemask notebook is not mentioned in this PR. When the requested changes have been made to the other notebooks, and they have been merged into the develop branch, please be sure to inform me (David Epstein), so that I can copy the changed cells into the tissuemask notebook.

Apologies for commenting on other matters than the first few cells.

**Pull request merged**